src/main/java/graphql/ErrorClassification.java

```diff
@@ -23,4 +23,21 @@ public interface ErrorClassification {
     default Object toSpecification(GraphQLError error) {
         return String.valueOf(this);
     }
+
+    /**
+     * This produces a simple ErrorClassification that represents the provided String.  You can
+     * use this factory method to give out simple but custom error classifications.
+     *
+     * @param errorClassification the string that represents the error classification
+     *
+     * @return a ErrorClassification that is that provided string
+     */
+    static ErrorClassification errorClassification(String errorClassification) {
+        return new ErrorClassification() {
+            @Override
+            public String toString() {
+                return errorClassification;
+            }
+        };
+    }
```

Comment:
I also backported this convenience method to make the backport easier - it's in v20 onwards